collectd-frontend/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM php:5.6-apache
```

Comment:
@esenti less configuration required. With apache we don't need to rewrite htaccess rules. Otherwise a fair point.